apps/core/components/passengerForm/PassengerForm.js

```diff
@@ -289,7 +290,7 @@ class PassengerForm extends React.Component<Props, State> {
             />
             <TextInput
               onChangeText={this.handleNameChange}
-              label="Given names"
+              label="First names"
```

Comment:
It seems to me that it should be "first name", without "s", what do you think?

Reply:
or maybe first name(s)

Reply:
👍

Reply:
done